src/Microsoft.OData.Client/DictionaryExtensions.cs

```diff
@@ -88,9 +88,13 @@ internal static class DictionaryExtensions
         }
 #else
         /// <summary>
-        /// Convenience function that wraps ConcurrentDictionary.TryAdd() to allow same signature as IDictionary
+        /// Convenience function that wraps ConcurrentDictionary.TryAdd() to allow same signature as IDictionary.
         /// </summary>
-        public static void Add<TKey, TValue>(this ConcurrentDictionary<TKey, TValue> self, TKey key, TValue value)
+        /// <typeparam name="TKey">The type of the key.</typeparam>
+        /// <typeparam name="TValue">The type of the value.</typeparam>        /// <param name="self">The ConcurrentDictionary to which to add the item.</param>

```

Comment:
> /// <param name="self">The ConcurrentDictionary to which to add the item.</param> [](start = 78, length = 82)

move to new line

Reply:
I think this is fixed?

---
In reply to: [258638342](https://github.com/OData/odata.net/pull/1204#discussion_r258638342) [](ancestors = 258638342)

Reply:
/// <param name="self">The ConcurrentDictionary ....
should be into new line. However, don't worry about it, it was fixed in the early PR. Please rebase to the latest codes.

Reply:
Ah, I see. I had already rebased, which is probably why I didn't see it.

---
In reply to: [262168586](https://github.com/OData/odata.net/pull/1204#discussion_r262168586) [](ancestors = 262168586)